src/components/Graph/Graph.tsx

```diff
@@ -0,0 +1,48 @@
+import Chart from 'react-google-charts';
+import { Container, GraphTitle, GraphWrapper } from './Graph.style';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Why @ts-ignore

Reply:
nice catch ! i did that to deploy yesterday and forgot to fix it